vpr/src/route/rr_graph.cpp

```diff
@@ -2960,19 +2979,10 @@ static void add_chain_node_fan_in_edges(RRGraphBuilder& rr_graph_builder,
                                                                       is_rr_sw_id,
                                                                       delay);
 
```

Comment:
Seems like you need a comment explaining why this data structure sometimes needs to be populated and sometimes doesn't.